File: lab 5/task2.cpp
#include <iostream>
#include <cmath>
using namespace std;
float calculate_power(float,float);
main()
{
    float number1,number2;
    cout<<"Enter the base number : ";
    cin>>number1;
    cout<<"Enter the exponent : ";
    cin>>number2;
    float power=calculate_power(number1,number2);
    cout<<number1<<" raised to the power "<<number2 <<" is : "<<power;


}
float calculate_power(float number1,float number2)
{
    float power=pow(number1,number2);
    return power;
}

File: python/task2.py
def number_length(number):
    count = 0
    if number == 0:
        count=1
        return count
    else:
        while(number!=0):
         count = count+1
         number=number//10
    return count    

number = int(input("Enter a number: "))  

length = number_length(number)
print("The length of number is: ",length)